test/Maestro.ScenarioTests/MaestroScenarioTestBase.cs

```diff
@@ -487,7 +487,9 @@ protected async Task AddDependenciesToLocalRepo(string repoPath, string name, st
     {
         using (ChangeDirectory(repoPath))
         {
-            await RunDarcAsync(["add-dependency", "--name", name, "--type", isToolset ? "toolset" : "product", "--repo", repoUri, "--version", "0.0.1"]);
+            await RunDarcAsync([
+                "add-dependency", "--name", name, "--version", "0.0.1", "--type", isToolset ? "toolset" : "product", "--repo", repoUri]
+                );
```

Comment:
Technically I think the indentation should go like this:

```suggestion
            await RunDarcAsync(
            [
                "add-dependency", "--name", name, "--version", "0.0.1", "--type", isToolset ? "toolset" : "product", "--repo", repoUri
            ]);
```

Reply:
Visual Studio wanted the indentation that is there currently (just formatted the file and it organized the block that way).

Do we have some style config that its not picking up which would instruct it to format the way you pointed out?

Reply:
Not sure. The feature ain't that old so maybe VS hasn't caught up yet?

Reply:
I'm on preview, but maybe; either way I change it to what you proposed